net.sf.eclipsecs.ui/.classpath

```diff
@@ -12,6 +12,12 @@
 			<attribute name="test" value="true"/>
 		</attributes>
 	</classpathentry>
+    <accessrules>
```

Comment:
What it this change for?

Reply:
My IDE build failed for these cases, thats why I had to add them